pkg/util/file.go

```diff
@@ -48,3 +49,13 @@ func WriteFileToLog(logger *capnslog.PackageLogger, path string) {
 
 	logger.Infof("Config file %s:\n%s", path, string(contents))
 }
+
+// PathToProjectRoot returns the path to the root of the rook repo on the current host.
+// This is primarily useful for tests.
+func PathToProjectRoot() string {
```

Comment:
While you're at it, there is a helper `FindRookRoot()` in the integration tests that could also be replaced by this cleaner approach. thanks!

Reply:
Should I move this to `tests/framework/utils` instead of `pkg/utils`?

Reply:
Sounds good since it's just test code if the dependencies work out